core/frontend/src/components/video-manager/VideoStreamCreationDialog.vue

```diff
@@ -356,6 +384,40 @@ export default Vue.extend({
     showDialog(state: boolean) {
       this.$emit('visibilityChange', state)
     },
+    selectedStreamType(endpoint: string): StreamType {
+      if (endpoint.startsWith('rtspu://')) return StreamType.RTSPU
+      if (endpoint.startsWith('rtsp://')) return StreamType.RTSP
+      if (endpoint.startsWith('rtspt://')) return StreamType.RTSPT
+      if (endpoint.startsWith('rtsph://')) return StreamType.RTSPH
```

Comment:
As a future note, we can use an Object.keys(StreamType).find((this_protocol) => new Url(endpoint).protocol.toUpper()) or something like that